qlib/data/dataset/processor.py

```diff
@@ -106,6 +106,22 @@ def process_inf(df):
 
         return replace_inf(df)
 
+class Fillna(Processor):
```

Comment:
It will better to support  `fields_group` like [other processors](https://github.com/you-n-g/qlib/blob/main/qlib/data/dataset/processor.py#L184) in case of use only want to fillna features.
It will better to support defining fillna value in the init functions 

I think we don't have to use datetime_groupby_apply